framework/console/controllers/CacheController.php

```diff
@@ -270,6 +270,12 @@ private function findCaches(array $cachesNames = [])
                 $caches[$name] = $component['class'];
             } elseif (is_string($component) && $this->isCacheClass($component)) {
                 $caches[$name] = $component;
+            } elseif (is_callable($component)) {
```

Comment:
@cebe `BaseYii::createObject()` [accepts](https://github.com/yiisoft/yii2/blob/master/framework/BaseYii.php#L350) not only instances of `Closure` but callable strings and arrays too. Are you sure we want to add this extra restriction?